schema/Retail/Status/on_status.ts

```diff
@@ -104,7 +104,7 @@ export const onStatusSchema = {
                       minLength: 1,
                     },
                   },
-                  required: ['id'],
+                  required: ['id', 'state'],
```

Comment:
pls incorporate time and invoice related checks as per the previous utility

Reply:
Can we do that in the next PR, as I have already changed this for flow related changes